Packages/com.unity.inputsystem/InputSystem/Events/InputEventTrace.cs

```diff
@@ -373,7 +373,8 @@ public void ReadFrom(Stream stream)
                         fixed(byte* tempBufferPtr = tempBuffer)
                         UnsafeUtility.MemCpy(tailPtr, tempBufferPtr, remainingSize);
 
-                        tailPtr += remainingSize;
+                        var alignedSizeInBytes = remainingSize.AlignToMultipleOf(4);
```

Comment:
- This needs many more tests, as it's not clear, for example, if allocated size of buffer is sufficient to hold all events
- Needs to use kAlignment constant instead of magic value